roles/container-engine/containerd/tasks/main.yml

```diff
@@ -101,7 +101,7 @@
 
 - name: Check if runc is installed
   stat:
-    path: /usr/sbin/runc
+    path: "{{ runc_binary }}"
```

Comment:
I added a default in my PR (https://github.com/kubernetes-sigs/kubespray/pull/5015/files#diff-6de31f202b896e5bda48a50cd465d94aR77). I think this is a good idea because otherwise Suse deployments will fail since the variables for suse doesn't contain a `runc_binary ` variable: https://github.com/kubernetes-sigs/kubespray/blob/master/roles/container-engine/containerd/vars/suse.yml

Ad soon as your PR is merged I would rebase mine.

Reply:
default has been already defined in defaults/main.yml.

You don't need redefine default.

Reply:
Ah, I missed this one, thanks !